admob/src/include/firebase/admob/types.h

```diff
@@ -159,8 +225,16 @@ struct AdRequest {
   ChildDirectedTreatmentState tagged_for_child_directed_treatment;
 };
 
+/// @deprecated
```

Comment:
Does our docs generation turn text after the \@deprecated into a note in the docs itself? If so, should we put something here?

Reply:
I tried that at first and the formatting was pretty horrible. The deprecated message appeared before all of the @brief text, so it didn't read well.  I have a screen shot I'll upload tomorrow.

Reply:
This is what the brief summary renders as if I create a message after the @deprecation tag:

```
/// @brief Iniitalizes AdMob via Firebase
///
/// <detailed description>
///
/// @deprecated <deprecation message>
Future<AdapterInitializationStatus> Initialize(const ::firebase::App& app, InitResult* init_result_out = nullptr);
```
<img width="868" alt="Screen Shot 2022-05-10 at 8 34 58 AM" src="https://user-images.githubusercontent.com/8961614/168282448-a0925668-4803-4a9f-a399-bc79abecbc59.png">

I personally prefer this format better.

Reply:
Hmm, is it able to link to firebase::gma namespace? (We might need to make some placeholder content like we did with Future on the gma side...)